src/main/kotlin/racingcar/application/race/factory/CarFactory.kt

```diff
@@ -7,12 +7,8 @@ import racingcar.domain.car.engine.MovingEngine
 class CarFactory(
     private val movingEngine: MovingEngine
 ) {
-    fun createByCount(count: Int): ParticipatingCars {
-        val cars = List(count) { Car(movingEngine) }
+    fun createByCarNames(carNames: List<String>): ParticipatingCars {
+        val cars = carNames.map { Car(movingEngine, it) }.toList()
         return ParticipatingCars(cars)
     }
```

Comment:
```suggestion
    fun createByCarNames(carNames: List<String>): ParticipatingCars {
        val cars = carNames.map { Car(movingEngine, it) }
        return ParticipatingCars(cars)
    }
```

`map` 의 반환타입이 이미 `List` 네요!

Reply:
그러네요 ㅎㅎㅎ [76931fc](https://github.com/next-step/kotlin-racingcar/pull/900/commits/76931fcc064f78656e6e562262d906eedcd0f889)에서 수정해보았습니다.